miso-web/src/main/webapp/scripts/box_visualization.js

```diff
@@ -56,12 +56,19 @@ var BoxPosition = function(opts) {
   };
 
   self.click = function(event) {
-    if (event.ctrlKey) {
+    if (isMultiSelectClick(event)) {
       self.controlClick();
     } else {
       self.normalClick();
     }
   };
+  
+  isMultiSelectClick = function(event) {
+    function isMac() {
+      return navigator.platform.indexOf('Mac') != -1;
+    }
+    return (isMac() && event.metaKey || !isMac && e.ctrlKey);
```

Comment:
Simplify to `return navigator.platform.indexOf('Mac') != -1 ? event.metaKey : e.ctrlKey;`
Also, you have `!isMac`, which checks the function, not the result.